### DISCUSSION TITLE
Missing 21.1.0 maven artifacts (keycloak-admin-ui, keycloak-account-ui)

### DISCUSSION THREAD

**Discussion by ssmtlbn** [keycloak/keycloak#19910] [Q&A] [answered]
The artifacts `org.keycloak:keycloak-admin-ui:21.1.0` and `org.keycloak:keycloak-account-ui:21.1.0` are missing in the maven repository. They are transitive dependencies of [`org.keycloak:keycloak-quarkus-server:21.1.0`](https://mvnrepository.com/artifact/org.keycloak/keycloak-quarkus-server/21.1.0).

Could they be published please?

**Comment by stustison** [comment]
Yes please, this keeps you from being able to rebuild the server in a separate project.

**Comment by vmuzikar** [comment, marked as answer]
Hello, this was a [bug](https://github.com/keycloak/keycloak/issues/19850) that will be fixed in 21.1.1. Sorry for the inconvenience.
